<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Part3Dialog</class>
 <widget class="QDialog" name="Part3Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>enter a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="a_value">
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>enter byte1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="byte1_value">
         <property name="maxLength">
          <number>1</number>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>enter byte2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="byte2_value">
         <property name="maxLength">
          <number>1</number>
         </property>
         <property name="placeholderText">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="text">
          <string>Binary representation of A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="binaryRepresentation">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="text">
          <string>A after swap</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="swap">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="text">
          <string>Binary representation of A after swap</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="binSwap">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Execute!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
